packages/grid/_modules_/grid/gridClasses.ts

```diff
@@ -389,6 +389,9 @@ export const gridClasses = generateUtilityClasses('MuiDataGrid', [
   'pinnedColumns',
   'pinnedColumns--left',
   'pinnedColumns--right',
+  'pinnedColumnHeaders',
```

Comment:
You also need to add these classes to the interface in this same file. There're should be docs changes in this PR.

Reply:
Is there a reason why we don't reuse the interface keys to generate the list of classes.

I found 3 elements in `gridClasses` which are not parts of the interface, and I don't know if it is expected or not :
- 'actionsCell'
- 'groupingCriteriaCell'
- 'groupingCriteriaCellToggle'

Reply:
I copied the same approach from MUI Core. I don't know how they check to avoid missing classes.